webapp/client/src/stories/FormFieldDropDown.stories.js

```diff
@@ -0,0 +1,106 @@
+import React from "react";
+
+import FormFieldDropDown from "../components/FormFieldDropDown";
+import StepForm from "../components/StepForm";
+import { Default as StepFormDefault } from "./StepForm.stories";
+
+export default {
+  title: "Form Fields/DropDown",
+  component: FormFieldDropDown,
+};
+
+const Template = (args) => (
+  <StepForm {...StepFormDefault.args}>
+    <FormFieldDropDown {...args} />
+  </StepForm>
+);
+
+export const Default = Template.bind({});
+Default.args = {
+  fieldId: "dropDown",
+  fieldName: "dropDown",
+  label: {
+    text: "Bundesland",
+  },
+  errors: [],
+  values: [
+    ["bw", "Baden-Württemberg"],
+    ["by", "Bayern"],
+    ["hh", "Hamburg"],
+    ["he", "Hessen"],
+  ],
+};
+
+export const WithDetails = Template.bind({});
+WithDetails.args = {
+  fieldId: "dropDown",
+  fieldName: "dropDown",
+  label: {
+    text: "Bundesland",
+  },
+  errors: [],
+  details: {
+    title: "Warum muss ich das auswählen?",
+    text: "Die Finanzämter sind nach Bundesländern sortiert.",
+  },
+  values: [
+    ["bw", "Baden-Württemberg"],
+    ["by", "Bayern"],
+    ["hh", "Hamburg"],
+    ["he", "Hessen"],
+  ],
+};
```

Comment:
You can simplify this as:

```suggestion
WithDetails.args = {
  ...Default.args,
  details: {
    title: "Warum muss ich das auswählen?",
    text: "Die Finanzämter sind nach Bundesländern sortiert.",
  },
};
```

Reply:
Thanks so much. This makes so much more sense